package.json

```diff
@@ -145,6 +145,30 @@
                 "vim.scroll": {
                     "type": "number",
                     "description": "Number of lines to scroll with CTRL-U and CTRL-D commands."
+                },
+                "vim.tabstop": {
```

Comment:
You don't know how happy it makes me to see we support all this stuff. My god, it's like we're actually making a real Vim plugin!!!